databricks_cli/dbfs/api.py

```diff
@@ -92,6 +92,9 @@ def get_status(self, dbfs_path):
         json = self.client.get_status(dbfs_path.absolute_path)
         return FileInfo.from_json(json)
 
+    def get_status_json(self, dbfs_path):
```

Comment:
It is not clean to expose both `get_status_json` and `get_status` in the api. Maybe `get_status` is sufficient?

Reply:
Maybe I'll use `_get_status_json` so it's not exposed in the API

Reply:
I want to get the actual JSON instead of the FileInfo object

Reply:
Actually, I'd like to keep it as `get_status_json` since I want it to be exposed in the API for purposes of stack deployment. I'll update the description of this for clarity.

Reply:
Hm yeah that seems cleaner. I added changes to the WorkspaceFileInfo class with a `json` property but haven't deleted the `get_status_json` function yet to keep the thread up. let me know how it looks!

Reply:
ok actually it's probably better to call get_status from the WorkspaceService `client` of `WorkspaceApi`.